app/app.go

```diff
@@ -159,7 +158,7 @@ var (
 
 	// ModuleEncodingRegisters keeps track of all the module methods needed to
 	// register interfaces and specific type to encoding config
-	ModuleEncodingRegisters = moduleMapToSlice(ModuleBasics)
+	ModuleEncodingRegisters = extractRegisters(ModuleBasics, appupgrade.TypeRegister{})
```

Comment:
followed through with @cmwaters 's idea to remove the module all together, and now we're just manually registering the upgrade module's types. 

This is necessary for the IBC module (https://github.com/celestiaorg/celestia-app/pull/1629#discussion_r1170264709)